cache-config/t3c-apply/util/util.go

```diff
@@ -432,3 +436,100 @@ func Touch(fn string) error {
 	myfile.Close()
 	return nil
 }
+
+func UpdateMaxmind(cfg config.Cfg) bool {
+
+	if cfg.MaxMindLocation == "" {
+		return false
+	}
+
+	// Dont update for report mode
+	if cfg.RunMode == t3cutil.ModeReport {
+		return false
+	}
+
+	// Split url, get filename
+	url, err := url.Parse(cfg.MaxMindLocation)
+	if err != nil {
+		log.Errorf("error parsing maxmind url: %v", err)
+		return false
+	}
+	urlpath := url.Path
+	fileName := path.Base(urlpath)
+
+	if fileName == "." || fileName == "/" {
+		log.Errorf("filename for maxmind from url invalid: %s", fileName)
+		return false
+	}
+
+	// Check if filename exists in ats etc
+	filePath := filepath.Join(cfg.TsConfigDir, "/", fileName)
```

Comment:
Nitpick: Join adds delimiters, the `/` is unnecessary. This will become `configdir//name`. Which works on Mac and Linux, POSIX specifies that duplicated slashes are collapsed. But it'll fail on Windows or other non `/` delimited systems.